docs/site/Testing-your-application.md

```diff
@@ -598,9 +585,85 @@ describe('ProductController (integration)', () => {
 
 ### Test your Services against real backends
 
-{% include content/tbd.html %}
+When integrating with other services (including our own microservices), it's
+important to verify that our client configuration is correct and the client
+(service proxy) API is matching the actual service implementation. Ideally,
+there should be at least one integration test for each endpoint (operation)
+consumed by the application.
 
-The initial beta release does not include Services as a first-class feature.
+To write an integration test, we need to:
+
+- Obtain an instance of the tested service proxy. Optionally modify the
+  connection configuration, for example change the target URL or configure a
+  caching proxy to speed up tests.
+- Execute service proxy methods and verify that expected results were returned
+  by the backend service.
+
+#### Obtain a Service Proxy instance
+
+In
+[./Calling-other-APIs-and-Web-Services.md#make-service-proxies-easier-to-test],
```

Comment:
Broken link here

Reply:
Oops, I forgot to commit and push fixes I made after I run Jekyll to render the updates. PTAL at 786ae96